demo/test-fpga-region.yaml

```diff
@@ -13,7 +13,7 @@ spec:
           [IPC_LOCK]
     resources:
       limits:
-        fpga.intel.com/arria10.dcp1.2-nlb3: 1
+        fpga.intel.com/arria10.dcp1.1-nlb3: 1
```

Comment:
Can you explain why we downgrade DCP version here? It makes sense to use the latest version available I believe.

Reply:
I focused on having a reliable K8s infrastructure. I have not upgraded all FPGA cards to DCP 1.2 so I will change accordingly the DCP version once the upgrading step is ready and tested